src/app/shared/auth.service.ts

```diff
@@ -5,9 +5,11 @@ import { HttpClient } from "@angular/common/http";
 })
 export class AuthService {
   constructor(private http: HttpClient) { }
-  readonly baseurl = '/api/employer/'
+  readonly baseurl = 'http://localhost:8080/v1/api/register/'
```

Comment:
I think you would get CORS policy error because of this while calling API's. 

use proxy.conf.json file